twilight-http/src/routing.rs

```diff
@@ -9,7 +9,10 @@ use twilight_model::id::{marker::RoleMarker, Id};
 #[non_exhaustive]
 pub enum Route<'a> {
     /// Route information to add a user to a guild.
-    AddGuildMember { guild_id: u64, user_id: u64 },
+    AddGuildMember {
+        guild_id: u64,
+        user_id: u64,
+    },
```

Comment:
this change seems unrelated?

Reply:
For some reason the formatter wants it to be in this way.